src/main/java/String18.java

```diff
@@ -0,0 +1,82 @@
+public class String18 {
+
+    public String addZero(String a, int countZero) {
+        StringBuffer strATemp = new StringBuffer(a);
+        for (int i = 0; i < countZero; ++i) {
+            a = strATemp.append("0").toString();
+            strATemp = new StringBuffer(a);
+        }
+        return a;
+    }
+
+    public int getSum(String a, String b) {
+
+        if (a.length() > b.length()) {
+            String temp = a;
+            a = b;
+            b = temp;
+        }
+
+        StringBuffer strResult = new StringBuffer("");
+
+        int indexA;
+        int indexB = b.length() - 1;
+
+        int surplus = 0;
+        int numberB = 0;
+        int numberA = 0;
+        int sum = 0;
+
+        for (indexA = a.length() - 1; indexA >= 0; --indexA) {
+
+            numberA = a.charAt(indexA) - '0';
+            numberB = b.charAt(indexB) - '0';
+
+            sum = numberA + numberB + surplus;
+
+            surplus = sum / 10;
+
+            strResult.append(sum % 10);
+            indexB--;
+        }
+
+        numberB = indexB >= 0 ? b.charAt(indexB--) - '0' : 0;
+        sum = surplus == 1 ? numberB + 1 : numberB;
```

Comment:
`surplus` chỉ có giá trị là 0 và 1 nên lệnh này tương đương với 

```suggestion
        sum = numberB + 1 ;
```

Reply:
done